packages/amplify-graphql-model-transformer/publish-notification-lambda/tsconfig.json

```diff
@@ -0,0 +1,109 @@
+{
```

Comment:
I would prefer not having so much commented out code or a quick follow up to enable these.

Reply:
This is the default tsc generated by 'tsc --init'. I can clean it up as a fast follow.